.travis.yml

```diff
@@ -132,6 +132,15 @@ jobs:
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=address,integer,undefined CC=clang CXX=clang++"
 
+    - stage: test
+      name: 'x86_64 Linux  [GOAL: install]  [no BIP70]'
+      env: >-
+        HOST=x86_64-unknown-linux-gnu
+        PACKAGES="clang llvm python3-zmq qtbase5-dev qttools5-dev-tools libssl1.0-dev libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev libqrencode-dev"
+        NO_DEPENDS=1
+        GOAL="install"
+        BITCOIN_CONFIG="--enable-zmq --disable-bip70 --with-incompatible-bdb --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=address,integer,undefined CC=clang CXX=clang++"
```

Comment:
I'd prefer to set `--disable-bip70` on one of the other bionic jobs that have wallet enabled.

Reply:
> @MarcoFalke It doesn't make sense to test this without the wallet. Is there a reason `x86_64 Linux [GOAL: install] [xenial] [no depends, only system libs, sanitizers: thread (TSan), no wallet]` needs to be wallet-less? Or where else would you want to add it?

Reply:
Maybe L133, L103, or L93?

Reply:
I think xenial has wallet disabled because the thread sanitizer yells at bdb